### DISCUSSION TITLE
ArchLinux 下无法正常显示缩略图，遇到 [thumbfast] 报错

### DISCUSSION THREAD

**Discussion by SWraven** [hooke007/mpv_PlayKit#491] [mpv 通用]
显示出第一张缩略图以后，鼠标在时间轴上移动不会再出现新的缩略图，命令行报错为 [thumbfast] Subprocess failed: init
通过调整quit_after_inactivity的数值可以勉强解决，但在缩略图多次刷新以后会无法再生成。
请问有无解决办法

**Comment by hooke007** [comment]
我没有Linux环境
